broker-plugins/jdbc-store/src/main/java/org/apache/qpid/server/store/jdbc/AbstractJDBCMessageStore.java

```diff
@@ -82,6 +82,8 @@
     private static final String XID_TABLE_NAME_SUFFIX = "QPID_XIDS";
     private static final String XID_ACTIONS_TABLE_NAME_SUFFIX = "QPID_XID_ACTIONS";
 
+    private static final int MAX_DELETE_BATCH_SIZE = 1000;
```

Comment:
Good idea. But to be consistent with the other system variables I propose "qpid.jdbcstore.inClauseMaxSize". I'm also changing the name to inClauseMaxSize because actually it's not really a batch here and it's not specific to the delete operation. WDYT?

Reply:
I agree that "qpid.jdbcstore.inClauseMaxSize" is much better name than "qpid.jdbcStoreMaxDeleteBatchSize". The variable name can be changed as well to inClauseMaxSize.

Reply:
Done